### DISCUSSION TITLE
Option for CCM web package to install IIS website with an alternate port binding

### DISCUSSION THREAD

**Discussion by Daxx23** [chocolatey/choco#2974] [Ideas] [answered]
I would like to see a parameter option for the CCM web package to specify an alternate IIS port binding.

Why is this feature request needed?
To prevent impact on already running IIS websites with a binding under port 80.
So Chocolatey Central Management can run on shared IIS webservers.

What is the benefit?
Being able to install the ChocolateyCentralManagement site in IIS under a different binding if that IIS webserver already has bindings on port 80 for different websites.

If you can provide specific examples, that would really help.
For customer A we choose to add the CCM role on an existing IIS webserver. After installation of the CCM web package, an IIS website already present on the IIS webserver was abruptly stopped as it also had a binding for port 80. After changing the IIS port binding on the ChocolateyCentralManagement IIS website to e.g. port 8080, I was able to start the other IIS website again and continue its function. Chocolatey Central Management dashboard was then accessible by adding the port 8080 to its internal webadres, e.g.; http://ccm:8080

**Comment by pauby** [comment, marked as answer]
This isn't the correct place to raise this issue.

Can I ask you to run `choco support` to see the options available to you and reach out to support?
